project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.3.12
+sbt.version = 1.5.0
```

Comment:
Scala 3 requires atleast SBT 1.5

Reply:
Play 2.9 requires at least sbt 1.9.6, not sure how this affect libraries such as this one though but upgrading to sbt 1.9.7 should be straightforward, isn't it?

Reply:
Ah good catch! Seems to compile fine and tests run, so lets bump it.

https://www.playframework.com/documentation/3.0.x/Highlights29#Scala-2.12,-sbt-0.13-and-Java-8-Support-discontinued